spec/octokit/client/repositories_spec.rb

```diff
@@ -390,22 +406,51 @@
         rules = {
           required_status_checks: {
             strict: true,
-            include_admins: true,
-            contexts: [] 
+            contexts: []
           },
+          enforce_admins: true,
+          required_pull_request_reviews: nil,
           restrictions: nil
         }
         branch = @client.protect_branch(@repo.full_name, "master", rules.merge(accept: preview_header))
 
-        expect(branch.required_status_checks.include_admins).to be true
+        expect(branch.required_status_checks.strict).to be true
         expect(branch.restrictions).to be_nil
         assert_requested :put, github_url("/repos/#{@repo.full_name}/branches/master/protection")
       end
+      it "protects a single branch with required_approving_review_count" do
+        rules = {
+          required_status_checks: {
+            strict: true,
+            contexts: []
+          },
+          enforce_admins: true,
+          required_pull_request_reviews: {
+            required_approving_review_count: 2
+          },
+        }
+        branch = @client.protect_branch(@repo.full_name, "master", rules.merge(accept: preview_header))
+
+        expect(branch.required_pull_request_reviews.required_approving_review_count).to eq 2
+        assert_requested :put, github_url("/repos/#{@repo.full_name}/branches/master/protection")
+      end
+
     end # .protect_branch
 
     context "with protected branch" do
       before(:each) do
-        @client.protect_branch(@repo.full_name, "master", accept: preview_header)
+        protection = {
+          required_status_checks: {
+            strict: true,
+            contexts: []
+          },
+          enforce_admins: true,
+          required_pull_request_reviews: nil,
+          restrictions: nil
+        }
+
+        @client.protect_branch(@repo.full_name, "master",
```

Comment:
We probably break long lines all over the place in this codebase, but my personal preference is long lines rather than broken lines, unless it gets excessive and ridiculous. Not something to change here, I'm just mentioning it in case everyone is like "oh, yes thank goodness, let's do that going forward".

Reply:
Got it, I've unwrapped it.